linux-cachyos-rc/PKGBUILD

```diff
@@ -556,7 +569,16 @@ prepare() {
 
 build() {
     cd "$_srcname"
-    make "${BUILD_FLAGS[@]}" -j"$(nproc)" all
+
+    # Use AutoFDO Profile named perf.afdo
+    # Maybe we can find a way to make this dyamic, but for now keep it hardcoded
+    # As soon LLVM 19 is in arch repository, we can also think about Propellor
+    if [ -n "$_autofdo_use" ]; then
+        make "${BUILD_FLAGS[@]}" CLANG_AUTOFDO_PROFILE=${srcdir}/perf.afdo -j"$(nproc)" all
```

Comment:
```suggestion
        make "${BUILD_FLAGS[@]}" CLANG_AUTOFDO_PROFILE="${srcdir}"/perf.afdo -j"$(nproc)" all
```

Reply:
BTW, you can make `CLANG_AUTOFDO_PROFILE` part of `BUILD_FLAGS`, so you don't have to do that check here and you just add a 
```
BUILD_FLAGS+=(CLANG_AUTOFDO_PROFILE="${srcdir}"/perf.afdo)
```
In a general `_autofod` check in `prepare()`.

Reply:
Thats a good idea